docker/openemr/flex-3.10/Dockerfile

```diff
@@ -38,6 +38,9 @@ RUN chmod 500 run_openemr.sh autoconfig.sh /root/unlock_admin.sh \
     && chmod 000 auto_configure.php /root/unlock_admin.php
 #fix issue with apache2 dying prematurely
 RUN mkdir -p /run/apache2
+#Copy dev tools alias to root
+COPY  utilities/devtools.sh /root/
```

Comment:
testing well except need to give execution permissions, so should follow above line with:
```
RUN chmod 500 /root/devtools.sh
```